### PR TITLE
Eve/Eve-7 Correct endcap calo tower offset

### PR DESCRIPTION
**Pull request by alja**

### CODE REVIEW

**Comment by alja** [comment]
@linev Please check

**Comment by linev** [comment]
@phsft-bot build

File: graf3d/eve/src/TEveCalo3DGL.cxx [hunk 1]
```diff
@@ -529,10 +529,7 @@ void TEveCalo3DGL::RenderEndCapCell(const TEveCaloData::CellGeom_t &cellData, Fl
 
    RenderBox(box);
 
-   if (z1 > 0)
-      offset += towerH * Cos(cellData.ThetaMin());
-   else
-      offset -= towerH * Cos(cellData.ThetaMin());
+   offset = towerH;
```
Comment (linev) [inline graf3d/eve/src/TEveCalo3DGL.cxx:532]: @alja, here `offset = towerH;`, but in eve7 it is `offset += towerH`
Is it correct?

**Comment by alja** [comment]
@linev Thanks, the correction is in the branch now. I did not noticed it because I made a test with only two stacks.

**Review by linev — APPROVED** [review]
Now looks better

**Comment by linev** [comment]
@phsft-bot build

**Comment by linev** [comment]
Jenkins finished on several platforms, can merge

**Pull request merged**